11-query/01-match-clause.md

```diff
@@ -414,10 +414,10 @@ GraqlGet query_a = Graql.match(
 [tab:end]
 </div>
 
-Running the above query on the `social_network` knowledge graph, returns the `post` concept type itself, as well as the concept types that subtype `post` directly (i.e. `media`, `comment`, `album` and `status-update`).
+Running the above query on the `social_network` knowledge graph, returns the `post` concept type itself, as well as the concept types that subtype `post` directlly (i.e. `media`, `comment`, `album` and `status-update`) or through any of its typetypes (i.e. `photo` and `video`).
```

Comment:
> directlly

is this a type?

Reply:
oh yes, it is! I must have written this before I had my morning coffee